### DISCUSSION TITLE
Support for Academia Militar MSc

### DISCUSSION THREAD

**Discussion by HBRebelo** [joaomlourenco/novathesis#417] [Q&A]
Hello,

I am a Professor at Academia Militar and would like to had support for the MSc thesis developed at this institution.

What are the required files and changes and would I be able to perform them?

Thank you very much.

Hugo Rebelo

**Comment by joaomlourenco** [comment]
@HBRebelo 
Thank you for your interest in the NOVAthesis template.
Just to make it clear, you would like to have support in the LaTeX-based template?

After lots of customisation for the different supported Schools, adding a new Schoo is no a very tough job, but having a good knowledge on LaTeX is a must!

The first step is to collect the “dissertation formatting rules” and the graphical elements (logos, etc).
Then, the suggestion is to “duplicate” the configuration from another School that is similar enough, and customise from there. I may try to help you in this task (if you want some help).
